templates/main.cf.j2

```diff
@@ -336,7 +345,7 @@ smtpd_sender_restrictions = {% for smtpd_sender_restriction in postfix_smtpd_sen
 #relayhost = uucphost
 #relayhost = [an.ip.add.ress]
 {% if postfix_relayhost is defined %}
-relayhost = {{ postfix_relayhost }}
+relayhost = [{{ postfix_relayhost }}]
```

Comment:
As above; please don't add the brackets, it's up to a user to select `host` or `mx-record`.